### ISSUE TITLE
Production mode build breaks mark inspector

### ISSUE THREAD

**Issue by kadamwhite** [uwdata/lyra#373]
When using `npm run build:prod` (as in #363) to generate a minified production build,

**Expected Behavior**

Making a mark and then clicking on it would display that mark's properties inspector as normal

**Actual Behavior**

Properties inspector does not load

![pipeline-issue](https://cloud.githubusercontent.com/assets/442115/15757151/4138bbf6-28d2-11e6-95c8-0e8543be2811.gif)

**Steps to reproduce behavior**
1. `npm run build:prod`
2. copy `build/`, `data/`, and `index.html` to a static file server
3. View Lyra on that server and observe properties inspector does not appear

Something's being clobbered somewhere by minification.

**Comment by deathbearbrown** [comment]
I KNOW WHAT IT IS...

To determine which thing to show we are relying on the constructor.name which is changed when you minimize everything
https://github.com/vega/lyra/blob/lyra2/src/js/components/InspectorSidebar.jsx#L35

https://github.com/vega/lyra/blob/lyra2/src/js/components/InspectorSidebar.jsx#L63-L67

So instead of "rect" the compiler gives you "r"

**Comment by arvind** [comment]
Good catch @deathbearbrown! One option is to switch to bracket notation (`Inspector['Line']`) but that feels fragile...

**Comment by deathbearbrown** [comment]
I really wish we weren't using stuff off the constructor, that's so gross

**Comment by kadamwhite** [comment]
Maybe each inspector constructor could give itself an ID or `.type`; [these lines](https://github.com/vega/lyra/blob/lyra2/src/js/components/InspectorSidebar.jsx#L63-L67) could then be changed to _e.g._

``` js
[
    require('./inspectors/Line'),
    require('./inspectors/Rect'),
    require('./inspectors/Symbol'),
    require('./inspectors/Text'),
    require('./inspectors/Area')
].forEach(function(InspectorCtor) {
    Inspector[InspectorCtor.type] = InspectorCtor;
});
```

Much preferable in my mind to explicitly give each constructor a "type" property than to rely on anything like `.constructor`

**Comment by arvind** [comment]
Fixed with #374. Thanks @deathbearbrown and @kadamwhite!
